net.sf.nootka.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "net.sf.nootka",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "nootka",
+  "branch": "1.4.5",
+
+  "rename-appdata-file": "nootka.appdata.xml",
+  "rename-desktop-file": "nootka.desktop",
+  "rename-icon": "nootka",
+
+  "finish-args":[
+      "--device=dri",
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--share=network",
+      "--filesystem=host",
```

Comment:
This is an excessive permission, what do you need all of host for?

Reply:
Changed to 'home'